openapi/swaggerHubUnresolved/ogc-api-tiles-opf-xmp-vt-more-1-collection.yaml

```diff
@@ -92,25 +92,25 @@ paths:
       responses:
         '200':
           description: |-
-            the URIs of all requirements classes supported by this API
+            the URIs of all conformance classes supported by this API
           content:
             application/json:
               schema:
                 $ref: 'https://api.swaggerhub.com/domains/UAB-CREAF/ogc-api-common/1.0.0#/components/schemas/confClasses'
               example:
                 conformsTo:              
                   # OGC API Common core consists on the landingPage, conformance
-                  - 'http://www.opengis.net/spec/ogcapi-common-1/1.0/req/core'
+                  - 'http://www.opengis.net/spec/ogcapi-common-1/1.0/conf/core'
                   # OGC API Common collections consists adds the capability to have collections
-                  - 'http://www.opengis.net/spec/ogcapi-common-1/1.0/req/collections'
+                  - 'http://www.opengis.net/spec/ogcapi-common-2/1.0/conf/collections'
                   # We need to be sure which ones are still valid when adopting OpenAPI.
                   - 'http://www.opengis.net/spec/ogcapi-features-1/1.1/req/oas30'
                   - 'http://www.opengis.net/spec/ogcapi-features-1/1.1/req/html'
                   - 'http://www.opengis.net/spec/ogcapi-features-1/1.1/req/geojson'
                   # OGC API Tiles core consists on the capability to serve a tiles from a collection 
-                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/req/core'
-                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/req/tms'
-                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/req/collections'
+                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/conf/core'
+                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/conf/tms'
+                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/conf/collections'
```

Comment:
This list of conformance classes is out of date...
The _Tiles_ conformance classes now are:
- tileset (we are talking about bringing back "core", by splitting everything but the "tileset metadata/description" out of tileset)
- tilesets-list (depends on: tileset)
- dataset-tilesets (depends on: tileset, tilesets-list)
- geodata-tilesets (depends on: tileset, tilesets-list)
- geodata-selection (we are discussing this `collections=` query parameter likely belongs in Common Part 2)

Reply:
Follow-on fix?

Reply:
Backed out the `ogcapi-tiles-1` changes